content/post/AG2020.en.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "2020 General Assembly Reviews"
+date: 2020-09-06
+draft: false
+tags: [general-assembly]
+---
+
+As every year, we organize our General Assembly. On September 5th, we met in the region of Nantes, with a few remote members at [BigBlueButton](https://bigbluebutton.org/) exhibition to review the past year and decide on the orientations of the association.
+
+### A busy but exhausting year for our volunteers
+
+This year has been very rich for Exodus Privacy. Our tools, whether it is the platform or the application, work well and are very used: The Android application has been downloaded more than 100,000 times from the Google Play Store and at the time of writing this blog, the platform contains more than 146,000 analyses of 77,452 applications. This year, we introduced the ability to analyze applications from [F-Droid](https://f-droid.org/) and added trackers to our knowledge base, which now contains more than 300 of those. This is something to be proud of!
+
+The goal of Exodus Privacy is to allow the greatest number of people to understand the issue of mobile phones tracking. For this purpose, several projects have been pursued by the association:
+
+* We have completed the redesign of the platform, thanks to the work of [Maiwann](https://www.maiwann.net/) and told it on [framablog](https://framablog.org/2019/11/29/collaborer-pour-un-design-plus-accessible-lexemple-dexodus-privacy/).
+* We have begun the project to redesign the application led by [Schoumi](https://mob-dev.fr/) following the recommendations of [Maiwann](https://www.maiwann.net/). A new version has been released this year with several improvements, but the work is still in progress.
+* We have set up [a translation platform](https://crowdin.com/profile/exodus-privacy) which today allows us to offer the platform in 5 languages and the application in 8 different languages.
+* We have launched a project for a pedagogical kit for digital mediators, which [we describe here](https://exodus-privacy.eu.org/fr/post/mednum/). We are waiting for an answer about its financing.
```

Comment:
Would be nice to have the mednum article translated as well (https://github.com/Exodus-Privacy/website/issues/109) and then fix the link to have it linking to the english version)

Reply:
I added it in #109 so it can't be forgotten once someone submit a PR and there will be still a valid link meanwhile.